### ISSUE TITLE
The output of Cutout is mask

### ISSUE THREAD

**Issue by Lmy0217** [aleju/imgaug#787]
![20210831165647](https://user-images.githubusercontent.com/16065878/131473738-babfb78a-b5cd-420d-9954-accd72dfa77f.png)

**Comment by ytliang97** [comment]
I encounter a similar problem. I use imgaug 0.4.0

I use the code below to augment my data. But I found out sometimes the image will turn black. I can only see a cutout area.

I notice that if I combine min-max normalize to 0-1 with `cutout(fill_mode='constant')`, I get this problem. If fill_mode changes to `'gaussian'`, it's ok. But I don't know why.


![image](https://user-images.githubusercontent.com/18035692/134958398-ef3afd67-bed7-43ab-904e-5be35f568020.png)
![image](https://user-images.githubusercontent.com/18035692/134958436-7a0e1850-7e8e-4825-a59d-39d64803a28b.png)
![image](https://user-images.githubusercontent.com/18035692/134958457-76c993bc-2098-4436-8db8-29e97dfa6c24.png)
![image](https://user-images.githubusercontent.com/18035692/134958477-e7cda2f5-581e-42af-aeb6-ac39263054c8.png)
![image](https://user-images.githubusercontent.com/18035692/134959130-3b7d2c4b-1752-4dd8-af31-eafdd6ea0706.png)



```python
from keras.preprocessing.image import ImageDataGenerator
from imgaug import augmenters as iaa
sometimes = lambda aug: iaa.Sometimes(0.5, aug)

def func_images(images, random_state, parents, hooks):
    result = []
    for image in images:
        X = (image - np.min(image)) / (np.max(image) - np.min(image))
        result.append(X)
    return result

# custom augmentation
normalizer = iaa.Lambda(
    func_images=func_images,
)

seed = np.random.randint(10000)
print(seed)
def combine_gen2(gen1, gen2, gen3):
    while True:
        yield(gen1.next(), gen2.next(), gen3.next())

datageno = ImageDataGenerator( # this is origin image
    #width_shift_range=0.25,
    #height_shift_range=0.25,
    #horizontal_flip=True,
    #rotation_range=15,
    #zoom_range=[0.5,1.0],
    #preprocessing_function=normalizer.augment_image
)

datagen = ImageDataGenerator( # this is for train image
    #width_shift_range=0.25,
    #height_shift_range=0.25,
    #horizontal_flip=True,
    rotation_range=15,
    #zoom_range=[0.5,1.0],
    preprocessing_function=seq.augment_image
)

datagen2 = ImageDataGenerator( # this is for train mask
    #width_shift_range=0.25,
    #height_shift_range=0.25,
    #horizontal_flip=True,
    rotation_range=15,
    #zoom_range=[0.5,1.0],
    fill_mode='nearest', 
    preprocessing_function=lambda x: np.where(x>0.5, 1, 0).astype(x.dtype)
)


# prepare iterator
xito = datageno.flow(bXtr, batch_size=32, seed=seed, shuffle=True)
xit = datagen.flow(bXtr, batch_size=32, seed=seed, shuffle=True)
yit = datagen2.flow(bYtr, batch_size=32, seed=seed, shuffle=True)
it = combine_gen2(xit,yit,xito)


for i in range(9):
    batch, batchy, batcho = it.__next__()
    image = batch[0]
    mask = batchy[0]
    oriimage = batcho[0]
    visualize(
        origin_image=oriimage,
        image=image,
#         mask=mask[:,:,1],
        hist_origin=oriimage,
        hist_image=image,
        
    )
```
